subiquity/common/resources.py

```diff
@@ -24,3 +24,26 @@
 
 def resource_path(relative_path):
     return os.path.join(os.environ.get("SUBIQUITY_ROOT", "."), relative_path)
+
+
+def get_users_and_groups(target, root):
```

Comment:
As this is impacting a common file between subiquity and system_setup, I suggest we split that in another PR (so, keep the duplicated code in this PR, just keep this commit staged) and we will review and get it reviewed by the subiquity team. What do you think?

Reply:
Can you elaborate more on this one, like, should I drop the commit from this PR and start another PR for this fix?

Reply:
Anyway, I have dropped the commit and apply the function directly in `get_userandgroups()`

Reply:
Yeah, let’s make that another PR once this is merged. Thanks!